runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersBrowseView/UsersBrowseView.utils.ts

```diff
@@ -0,0 +1,29 @@
+import useTranslations from '#src-app/hooks/useTranslations';
+import { UsersTab } from '#src-app/utils/users-tab';
+
+export enum DefaultValue {
+    PAGE_SIZE = 10,
+    PAGE = 0,
+}
+
+export const ROWS_PER_PAGE = [10, 20, 30];
+
+interface UsersTabsProps {
+    value: string,
+    label: string
+}
```

Comment:
props are what you pass, but here you're returning this object rename to UsersTabsHookResult or name it accordingly to later use (like e.g. AcceptedUserTableCell)